src/test/java/uk/gov/companieshouse/search/api/controller/AlphabeticalSearchControllerTest.java

```diff
@@ -2,20 +2,16 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.mockito.Mockito.doReturn;
-import static org.mockito.Mockito.when;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
```

Comment:
I think your intelij settings need fixing, as seems to be condensing imports into wildcard imports which is a potential security issue in the future.